app/views/logged_out.erb

```diff
@@ -0,0 +1,24 @@
+<div class="panel">
+  <div class="panel-heading">
+    <h2 class="panel-title" >Join In?</h2>
+  </div>
+    <form action='/users' method='post' class='logged_out'>
+      <input type='text' name='name' placeholder='Name'><br>
```

Comment:
Bear in mind placeholders only work back to IE10 - not IE9 or IE8 (or Opera).

Also placeholders cannot (in the main) be read by screen readers. Your disabled visitors have no idea how to log in to your site.

Solve this with

```erb
<label for="name">Name</label>
```